boards/arm/nrf52_adafruit_feather/doc/nrf52_adafruit_feather.rst

```diff
@@ -0,0 +1,174 @@
+.. _nrf52_adafruit_feather:
+
+nRF52 Adafruit Feather
+######################
+
+Overview
+********
+
+Zephyr applications use the nrf52_adafruit_feather board configuration
+to run on the nRF52 Adafruit Bluefruit Feather hardware. It provides
+support for the Nordic Semiconductor nRF52832 ARM Cortex-M4F CPU and
+the following devices:
+
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* UART
+* GPIO
+* FLASH
+* RADIO (Bluetooth Low Energy)
+* Segger RTT (RTT Console)
+
+.. figure:: img/nRF52Feather.png
+     :width: 442px
+     :align: center
+     :alt: nRF52 Adafruit Feather Board
+
+     nRF52 Adafruit Feather Board (Credit: Adafruit)
+
+More information about the board and its features can be found at the
+`Adafruit Feather nRF52 Bluefruit Learning Guide`_. The `Nordic Semiconductor Infocenter`_
+contains the processor's information and the datasheet.
+
+Hardware
+********
+
+- nRF52832 ARM Cortex-M4F processor at 64 MHz
+- 32.768 kHz crystal oscillator
+- 512 KiB flash memory and 64 KiB of SRAM
+- Battery connector and charger for 3.7 V lithium polymer batteries
+- Charging indicator LED
+- 2 User LEDs
+- Reset button
+- SWD connector
+- USB serial converter
+
+Supported Features
+==================
+
+The nRF52 Adafruit Feather board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| NVIC      | on-chip    | nested vectored      |
+|           |            | interrupt controller |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial port          |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTT       | on-chip    | console              |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+
+Connections and IOs
+===================
+
+The `Adafruit Feather nRF52 Bluefruit Learning Guide`_ has detailed
+information about the board including `pinouts`_ and the `schematic`_.
+
+LED
+---
+
+* LED0 (red) = P0.17
+* LED1 (blue) = P0.19
+
+Push buttons
+------------
+
+* DFU = SW0 = P0.20
+* RESET = SW1 = P0.21/reset
+
+Programming and Debugging
+*************************
+
```

Comment:
Can you please add a section here describing what needs to be done and bought in order to use Segger with this board?